packages/e2e-tests/specs/experiments/multi-entity-saving.test.js

```diff
@@ -101,14 +117,39 @@ describe( 'Multi-entity save flow', () => {
 				await assertMultiSaveEnabled();
 			} );
 
-			it( 'Clicking should open modal with boxes checked by default', async () => {
+			it( 'Should only have save panel a11y button active after child entities edited', async () => {
+				await assertExistance( publishA11ySelector, false );
+				await assertExistance( saveA11ySelector, true );
+				await assertExistance( publishPanelSelector, false );
+				await assertExistance( savePanelSelector, false );
+			} );
+
+			it( 'Clicking should open panel with boxes checked by default', async () => {
 				await page.click( savePostSelector );
+				await page.waitForSelector( savePanelSelector );
 				await assertAllBoxesChecked();
 			} );
 
-			it( 'Saving should result in items being saved', async () => {
+			it( 'Should not show other panels (or their a11y buttons) while save panel opened', async () => {
+				await assertExistance( publishA11ySelector, false );
+				await assertExistance( saveA11ySelector, false );
+				await assertExistance( publishPanelSelector, false );
+			} );
+
+			it( 'Publish panel should open after saving, no other panels (or their a11y buttons) should be present', async () => {
+				// Save entities and wait for publish panel.
 				await page.click( entitiesSaveSelector );
+				await page.waitForSelector( publishPanelSelector );
+
+				await assertExistance( publishA11ySelector, false );
+				await assertExistance( saveA11ySelector, false );
+				await assertExistance( savePanelSelector, false );
```

Comment:
Shouldn't we also check that the correct panel is open here?

Reply:
We are with the `waitForSelector`.  If the correct panel was not open, that would timeout and the test would fail.